course/src/isometry.cpp

```diff
@@ -10,9 +10,146 @@
 namespace ekumen {
 namespace math {
 
-/*
- * your code here
- */
+namespace {
+
+Vector3 translation_vector_operation(const Matrix3& matrix,
+                                     const Vector3& first_vector,
+                                     const Vector3& second_vector) {
+  Vector3 aux;
+  aux = matrix * first_vector + second_vector;
+  return aux;
+}
+
+}  // namespace
+
+
+Isometry Isometry::fromTranslation(const Vector3& r_vector) {
+  Isometry aux;
+  aux.translation_vector = r_vector;
+  return aux;
+}
+
+Vector3 Isometry::translation() const {
+  Vector3 aux{translation_vector};
+  return aux;
+}
+
+Vector3 Isometry::transform(const Vector3& r_vector) const {
+  Vector3 aux;
+  aux = (*this) * r_vector;
+  return aux;
+}
+
+Isometry Isometry::inverse() const {
+  double deter = rotation_matrix.det();
```

Comment:
nit: make it `const`. And `det` sounds better. :man_dancing:

Reply:
Addressed.